rs/tests/execution/BUILD.bazel

```diff
@@ -130,10 +130,7 @@ system_test(
 
 system_test_nns(
     name = "canister_migration_test",
-    enable_mainnet_nns_variant = False,
```

Comment:
TBH, I am not sure what this does, but it seems orthogonal to the main purpose of this PR?

Reply:
nvm. This enables more tests.